gcloud-java-examples/README.md

```diff
@@ -53,39 +53,39 @@ To run examples from your command line:
     ```
     Then you are ready to run the following example:
     ```
-    mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.BigQueryExample" -Dexec.args="create dataset new_dataset_id"
-    mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.BigQueryExample" -Dexec.args="create table new_dataset_id new_table_id field_name:string"
-    mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.BigQueryExample" -Dexec.args="list tables new_dataset_id"
-    mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.BigQueryExample" -Dexec.args="load new_dataset_id new_table_id CSV gs://my_bucket/my_csv_file"
-    mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.BigQueryExample" -Dexec.args="query 'select * from new_dataset_id.new_table_id'"
+    mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.bigquery.BigQueryExample" -Dexec.args="create dataset new_dataset_id"
```

Comment:
This can be done in another PR, but I wonder if there's a way to shorten the main class specification.  Since all the files use the same pom, it isn't as easy as defining the main class in the exec:java plugin section of the pom.  But I do think it would be a better getting-started experience if we could make it shorter.